tests/InputFiles/Poisson/ProductOfSinusoids3D.yaml

```diff
@@ -5,6 +5,7 @@ Executable: SolvePoisson3D
 Testing:
   Check: parse;execute_check_output
```

Comment:
The 1D version of this input file is used in `Test_ValidateInputFile.py`. Gonna have to pick a new executable to use in that test.

Reply:
Done